xbmc/video/windows/GUIWindowVideoNav.h

```diff
@@ -49,7 +49,7 @@ class CGUIWindowVideoNav : public CGUIWindowVideoBase
   virtual bool OnAction(const CAction &action) override;
   virtual bool OnMessage(CGUIMessage& message) override;
 
-  virtual void OnItemInfo(const CFileItem& fileItem, ADDON::ScraperPtr &info) override;
+  virtual void OnItemInfo(CFileItem* fileItem, ADDON::ScraperPtr &info) override;
```

Comment:
I'll experiment with changing them back and also change the music part. If it works, no harm done, if not we need to trace the error.
See https://github.com/Razzeee/xbmc/pull/3#issuecomment-233133597 for a log excerpt